lastools-el7/fpm.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+version=170414
```

Comment:
Oh I didn't realise thats the version that the binary returned. In which case I agree. I thought it was just chosen by John as the date of the last changes in the CHANGES file.